api/src/pcapi/scripts/rebuild_staging/anonymize.sql

```diff
@@ -286,7 +286,20 @@ WHERE reason is not null
 
 TRUNCATE TABLE activity;
 TRUNCATE TABLE user_session;
+
+-- Avoid synchronization with providers
 UPDATE venue_provider SET "isActive" = false;
+-- Deactivate the related offers
+-- Analysis of the query: https://passculture.atlassian.net/browse/PC-28443
+-- TL;DR: It takes about 100 seconds, and uses the indexes. No seq scan of the offer table is happening.
+-- This is the best we can do to avoid a full table scan.
+UPDATE offer
+SET "isActive" = FALSE
+FROM venue_provider
+JOIN provider ON provider.id = venue_provider."providerId"
+WHERE venue_provider."venueId" = offer."venueId"
+  AND provider."localClass" = in ('CDSStocks','BoostStocks','CGRStocks','EMSStocks')
```

Comment:
T'es sûr de la syntaxe `= in` :o

Reply:
meilleur question est ce qu'on peux tester l'anonymization avant la prod??

Reply:
Rien ne nous empêche de tester en local en effet.

Reply:
Ah my bad
Yes je vais re-lancer la dernière version de la query en staging 👍